extensions/vertx/runtime/src/main/java/io/quarkus/vertx/core/runtime/config/AddressResolverConfiguration.java

```diff
@@ -26,4 +28,15 @@
     @ConfigItem(defaultValue = "0")
     public int cacheNegativeTimeToLive;
 
+    /**
+     * The maximum number of queries to be sent during a resolution.
+     */
+    @ConfigItem(defaultValue = "4")
+    public int maxQueries;
+
+    /**
+     * The duration after which a DNS query is considered to be failed.
+     */
+    @ConfigItem(defaultValue = "PT5S")
```

Comment:
```suggestion
    @ConfigItem(defaultValue = "5")
```

Reply:
@machi1990 May I ask what your concerns are regarding the usage of `PT5S`?

Reply:
@sophokles73 sorry, I should have added the description. The duration converter we have in place is capable of interpreting `5` or `5S`. In my opinion, the two are more user friendly when the config documentation is generated because the default value will be generated as `5S` which reads easier than `PT5S`.
See an example of the generated config here https://quarkus.io/guides/redis#quarkus-redis-client_quarkus.redis.timeout

Reply:
Makes sense, thanks for the explanation. I will push a corresponding change ...